spec/features/editing_a_cma_case_spec.rb

```diff
@@ -124,6 +124,15 @@
       assert_publishing_api_put_content(content_id, request_json_includes(changed_json))
     end
 
+    scenario "date values display" do
+      visit "/cma-cases/#{content_id}"
+
+      expect(page).to have_content("2014")
```

Comment:
Could we use more specific selectors here? Something like:
```ruby
expect(page).to have_selector("input[name=[cma_case]opened_date(1i)]", with: "2014")
```
Also I think the labels for these inputs all have the wrong 'for' attribute when you inspect them